.github/workflows/diff_report.yml

```diff
@@ -35,7 +35,16 @@ jobs:
       branch: ${{ steps.branch.outputs.ref }}
       
     steps:
-    
+     - uses: khan/pull-request-comment-trigger@master
+       name: React with rocket on run
+       with:
+         trigger: 'GitHub, generate report'
```

Comment:
I do not like duplicate of trigger words.
please make `         trigger: ' '` (trigger by space) as all our trigger phrases has spaces and all we need is to run `pull-request-comment-trigger` all the time. I wish it could allow triggering all the time.

https://github.com/Khan/pull-request-comment-trigger

Reply:
' ' Does not work as it is considered by it a null character. 
Error log with space :- 
``` No `trigger` input given, aborting.```

Reply:
We can change it something like ',' or any word or letter.

Reply:
please do `,`

Reply:
Solved